frontend/js/src/artist-similarity/src/Graph.tsx

```diff
@@ -0,0 +1,112 @@
+import React, {useState} from 'react';
```

Comment:
I think this file/module should be called SimilarArtistsGraph

Reply:
Renamed to SimilarArtistsGraph